.github/scripts/run_with_env_secrets.py

```diff
@@ -56,6 +56,10 @@ def main():
         except KeyError as e:
             print(f"Could not set {senv} from secret {sname}: {e}")
 
+    env_file = f"{ os.environ.get('RUNNER_TEMP', '') }/github_env_{ os.environ.get('GITHUB_RUN_ID', '') }"
+    if f"{ os.environ.get('ROCM', '') }" == "true":
+        env_file = f"/tmp/github_env_{ os.environ.get('GITHUB_RUN_ID', '') }"
+
```

Comment:
This shouldn't be needed if we update `setup-rocm.yml` to export all the required env vars to `${RUNNER_TEMP}/github_env_${GITHUB_RUN_ID}` like in [actions/setup-linux/action.yml](https://github.com/pytorch/test-infra/blob/main/.github/actions/setup-linux/action.yml)